React/Views/RCTMap.m

```diff
@@ -155,4 +156,45 @@ - (void)setAnnotations:(RCTPointAnnotationArray *)annotations
   self.annotationIds = newIds;
 }
 
+- (void)setPolylines:(RCTPolylineArray *)polylines
+{
+  NSMutableArray *newPolylineIds = [NSMutableArray new];
+  NSMutableArray *polylinesToDelete = [NSMutableArray new];
+  NSMutableArray *polylinesToAdd = [NSMutableArray new];
+
+  for (RCTPolyline* polyline in polylines) {
+    if (![polyline isKindOfClass:[RCTPolyline class]]) {
```

Comment:
This check is not necessary, since it's already enforced by the RCTConvert function.